src/session/session-utils.c

```diff
@@ -164,6 +164,24 @@ write_control_end (void)
   auth_msg_size = 0;
 }
 
+__attribute__((__noreturn__)) void
+exit_init_problem (const char *problem,  const char *message)
+{
+  char *payload = NULL;
+
+  debug ("writing init problem %s message %s", problem, message);
+
+  if (asprintf (&payload, "\n{\"command\":\"init\",\"version\":1,\"problem\":\"%s\",\"message\":\"%s\"}",
+                problem, message) < 0)
```

Comment:
Sure hope we never try to put double quotes or backslashes in those messages...

Reply:
See thread below.